utilities/cli/tests/test_module_template_pkg.py

```
@@ -0,0 +1,329 @@
+
+def _cookiecutter_available() -> bool:
+    try:
+        import cookiecutter  # noqa: F401
+
+        return True
+    except ImportError:
+        return False
+
+
+def _run(cmd: list, *, env: dict | None = None) -> subprocess.CompletedProcess:
+    return subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)